frb_rust/src/misc/logging.rs

```diff
@@ -0,0 +1,233 @@
+#[macro_export]
+macro_rules! enable_frb_logging {
+    () => {
+#[allow(clippy::crate_in_macro_def)]
+use crate::frb_generated::StreamSink;
+use flutter_rust_bridge::frb as frb_4_log;
```

Comment:
nit: maybe just do not `use` anything, since otherwise the user will get confused after executing this macro. Instead, just write down the super-long name directly (a bit ugly but acceptable).

Reply:
done :)